subprojects/core/src/main/java/org/gradle/process/internal/daemon/WorkerDaemonClient.java

```diff
@@ -13,26 +13,24 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.gradle.api.internal.tasks.compile.daemon;
+package org.gradle.process.internal.daemon;
 
 import org.gradle.internal.concurrent.Stoppable;
-import org.gradle.language.base.internal.compile.CompileSpec;
-import org.gradle.language.base.internal.compile.Compiler;
 
-class CompilerDaemonClient implements CompilerDaemon, Stoppable {
+class WorkerDaemonClient implements WorkerDaemon, Stoppable {
```

Comment:
I'm a bit confused by this name. It sounds to me like "someone who is using a worker daemon", but it is actually the implementation of the worker daemon interface.